### ISSUE TITLE
Getting error while executing do_nms(boxes, 0.5) cell

### ISSUE THREAD

**Issue by vishalkr058** [dsbyprateekg/YOLOv4#6]
I am using my own custom YOLOv4 weights file which i trained to detect 3 objects in the images.
When i am trying to execute cell where we have to call **do_nms(boxes, nms_thresh)** function then it is throwing error **ZeroDivisionError: float division by zero** . Here is a screenshot of the error 
<img width="511" alt="error" src="https://user-images.githubusercontent.com/22008102/121223368-1573c600-c8a5-11eb-9d14-094345ad0150.PNG">

**Comment by vishalkr058** [comment]
@dsbyprateekg if you could help me in this issue

**Comment by dsbyprateekg** [comment]
@vishalkr058 It seems somehow the value of union at line no 25 is zero which caused this error. To handle this you can put this line number inside a try block and in except block you can handle this error like this-
`try:
    return float(intersect)/union
except ZeroDivisionError:
    return 0`

Another way is to put a not null check
`if union != 0:
    return float(intersect)/union
else:
    return 0`

**Comment by vishalkr058** [comment]
@dsbyprateekg I tried to use ur method to handle the error and it worked but seems like the output image at the end is not showing properly as shown below

<img width="845" alt="output_image" src="https://user-images.githubusercontent.com/22008102/121328893-5665ec00-c932-11eb-8bcc-216b31e6a99a.PNG">

I think there are some issues occurred while computing the yolov4 weight file, so, i checked and found out that there are errors showing in the cell where we have to **Load and Compute the YOLOv4 weights** and in the cell **Compute the Yolo layers** . below are the screenshot of the errors occurs while computing

<img width="844" alt="load and compute yolo weight" src="https://user-images.githubusercontent.com/22008102/121329537-ec9a1200-c932-11eb-8aae-feb13f1e88df.PNG">
<img width="817" alt="compute yolo layer" src="https://user-images.githubusercontent.com/22008102/121329574-f3c12000-c932-11eb-8c6d-cc5e940e834a.PNG">

**Comment by dsbyprateekg** [comment]
@vishalkr058 have you trained your dataset on tiny version of yolo?
Also please confirm you are using following version of TF and keras-
TensorFlow version is: 2.2.0
Keras version is: 2.3.0-tf

**Comment by vishalkr058** [comment]
@dsbyprateekg No, I haven't trained my dataset on tiny version of yolo

Here are the version of TF and Keras i am using:
Tensorflow version: 2.3.2
Keras version: 2.4.3

**Comment by dsbyprateekg** [comment]
@vishalkr058 this notebook was tested on below version so your issue may be due to the version upgrade.
Please install the below ones in a separate env and then test your model conversion-
TensorFlow version is: 2.2.0
Keras version is: 2.3.0-tf

**Comment by vishalkr058** [comment]
@dsbyprateekg ok, i will try again and also Keras version that u mentioned as 2.3.0-tf , so, is it same as version 2.3.0 ?

**Comment by dsbyprateekg** [comment]
yes, it's same. The notebook was tested on Colab env and there it was showing 2.3.0-tf.

**Comment by vishalkr058** [comment]
@dsbyprateekg Just like yolo .weights model has config file in the .cfg format so, in the same way how can i generate .pbtxt file from the saved .pb model ?

**Comment by dsbyprateekg** [comment]
@vishalkr058 I don't know about this. May I know why do you want this?

**Comment by vishalkr058** [comment]
@dsbyprateekg actually the purpose of converting my custom yolo weight file to tensorflow is to use this tensorflow model with OpenVINO toolkit in order to create application. And In Openvino I need pbtxt file also along with saved tensorflow model(pb file) to generate intermediate representation(IR) thru model optimizer.

**Comment by dsbyprateekg** [comment]
@vishalkr058 I suggest you to follow this post for YOLOv4+OpenVINO integration-
[https://devmesh.intel.com/projects/openvino-yolov4-49c756](url)
